tests/ci/integration/bind9_patch/bind-fix.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/tests/isc/Makefile.am b/tests/isc/Makefile.am
+index 5cdd915..6ee1935 100644
+--- a/tests/isc/Makefile.am
++++ b/tests/isc/Makefile.am
+@@ -115,10 +115,12 @@ proxyheader_test_SOURCES =	\
+ 	proxyheader_test_data.h
+ 
+ proxystream_test_CPPFLAGS =	\
+-	$(AM_CPPFLAGS)
++	$(AM_CPPFLAGS)			\
```

Comment:
Do you have any idea how this works with OpenSSL and Bind right now?

Reply:
Don't we install our package config?

Reply:
Yeah we install our binaries with pkgconfig, but without the flags here the test binary isn't finding the right artifacts.
https://github.com/aws/aws-lc/blob/ddc449d4f17ea2621a2e8d36138f2525e05a86a9/CMakeLists.txt#L1137-L1139

Reply:
Is there a built in way to tell package config to look for other package config in a different location?

Reply:
Yes, the PKG_CONFIG_PATH environment variable was pkgconfig's own recommendation to get around this. I added the comment based off of that.
We use this in all our integration ci scripts that use pkgconfig.